### PR TITLE
如何做出一個好的 NodeJS 模組？

### PR DESCRIPTION
**Pull request by tigercosmos**
new post!
r? @aszx87410 @kdchang

### CODE REVIEW

**Comment by kdchang** [comment]
Hi @tigercosmos ,

Thanks for your post. happy new year :)

@ArvinH @aszx87410 @Po-Jen 
Please help to code review. Thank you :)

File: source/_posts/how-to-make-a-good-node-module.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+date: 2018-02-14 13:55:57
+tags: nodejs, npm, module, tutorial
+author: tigercosmos
+---
+
+## 前言
+
+這篇文章需要知道什麼是 [NodeJS](https://www.w3schools.com/nodejs/default.asp)，簡單來說 NodeJS 讓 Javascript 可以變成一個後端的語言，而不是僅限瀏覽器才能解讀的前端語言。如果你還沒用過 NodeJS，首先需要安裝他，Windows 和 MacOS 的用戶可以直接上[官網](https://nodejs.org/en/)下載，Linux 的用戶推薦使用 [NVM](https://github.com/creationix/nvm) 安裝。
+
+回想我們一開始學 C++ 的時候，如果想要用數學的函式，必須引入 `cmath` 函式庫（Library）才能呼叫 `sin`、`cos` 等函示。而我們在寫 NodeJＳ 的時候，一樣可能會需要引入函式庫加入主程式，在 NodeJS 中我們稱為模組（Module）。寫一個模組雖然很簡單，但若是想要讓自己寫的模組給更多人使用，就要注意一些小細節，讓我娓娓道來。
```
Comment (aszx87410) [inline source/_posts/how-to-make-a-good-node-module.md]: > 才能呼叫 `sin`、`cos` 等函示

=> 函式

Comment (aszx87410) [inline source/_posts/how-to-make-a-good-node-module.md]: > 而我們在寫 NodeJＳ 的時候

NodeJS，S 不小心用成全型的了XD

**Comment by aszx87410** [comment]
Hi, @tigercosmos 
Happy new year!
I leave some comments and it's mostly about typo. Apart from this, it looks good to me. Thanks for sharing!

File: source/_posts/how-to-make-a-good-node-module.md (hunk 1 above)
Comment (tigercosmos) [reply source/_posts/how-to-make-a-good-node-module.md]: 這樣都看出來了！！厲害XD

**Comment by tigercosmos** [comment]
I have fixed them. Happy Chinese new year~

**Review by kdchang — APPROVED** [review]
LGTM. Thank you :)

**Comment by kdchang** [comment]
Hi @tigercosmos 

I have added you to our team slack. Please check it. thank you:)

BTW, congratulation on winning ironman prize !

**Comment by tigercosmos** [comment]
Thanks :)

**Pull request merged**